facebookresearch/sumo-challenge
# Update setup.py to install sumo as a package

Steps to install it locally:
1. Clone the repository and switch to this PR
2. Go to `sumo-api` directory
3. Run `pip install -r requirements.txt`
4. Run `pip install .`

The sumo package will be installed locally.

To Verify it:
1. Open python shell
2. Run `import sumo`
The command should run properly without any error.
